### DISCUSSION TITLE
Why this application is not working on macOS Sonoma?

### DISCUSSION THREAD

**Discussion by MSamiulHasnat** [noah-nuebling/mac-mouse-fix#687] [Questions] [answered]
Device - Mabook Air 2020 with 8GB of RAM
OS - macOS Sonoma
Application Version - Both v2 and v3
Mac Mouse Fix app is not running on my machine. It opens but doesn't enable.

**Comment by garethbhughes** [comment]
Same issue on an M1 Macbook Pro. I installed v3 and it worked great at first. Then I disabled it while testing other mouse helpers to compare. Now I can't reenable it -- the "enable" button does nothing.

I've already uninstalled and reinstalled the app, rebooted the computer, and removed and readded the permissions, all with no effect.

**Reply by orkanksi** [reply]
I installed the 2.0 then disabled and wanted reenable, didn't work. Updated to Beta, paid for activation, still cannot enable. Also changed accessibility and security settings no avail. Is there a way to "clean uninstall" so I can maybe reinstall? 

System: M1 Sonoma 14.1.1. wireles generic mouse

**Comment by noah-nuebling** [comment, marked as answer]
Hey guys, please continue discussion on this in the linked issue #688. Thank you.
